pkg/broker/handler/pool/testing/helper.go

```diff
@@ -373,6 +374,35 @@ func (h *Helper) VerifyNextTargetEvent(ctx context.Context, t *testing.T, target
 	h.VerifyAndRespondNextTargetEvent(ctx, t, targetKey, wantEvent, nil, http.StatusOK)
 }
 
+// VerifyNextTargetEventWithInfiniteDelay verifies the next event the subscriber receives
+// but not respond a success infinitely.
+func (h *Helper) VerifyNextTargetEventAndDelayResp(ctx context.Context, t *testing.T, targetKey string, wantEvent *event.Event, delay time.Duration) {
+	t.Helper()
```

Comment:
Can you consolidate this with `VerifyAndRespondNextTargetEvent`? `VerifyAndRespondNextTargetEvent` basically calls this with delay=0